catalog-info.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component                                       # Alterar para 'Component', 'API' ou 'Resource'
+metadata:
+  name: gupy-api-factory
+  description: None
+  annotations:
+    github.com/project-slug: gupy-io/gupy-api-factory
+    backstage.io/techdocs-ref: dir:. 
+spec:
+  type: service                                       # Alterar para 'service', 'website' ou 'library'
+  owner: catalogo-incompleto                          # Confome nome no GitHub ('qualificacao', 'vagas', 'candidates', por exemplo)
```

```suggestion
  owner: engenharia                         # Confome nome no GitHub ('qualificacao', 'vagas', 'candidates', por exemplo)
```

everybodyyy yeaaaahhhhh rcok your bodyyy yeaah